setup.py

```diff
@@ -19,9 +19,20 @@
 tests_require = [
     'pytest',
     'mock',
-    'coverage < 4',
+    'coverage < 4'
 ]
 
+install_requires = [
+    'lxml',
+    'click',
+    'boto3',
+    'requests',
+    'configparser'
+]
+
+if system() == 'Windows':
+    install_requires.append('requests-negotiate-sspi')
```

Comment:
There's another Negotiate module out there that uses the Linux Kerberos GSSAPI, but it doesn't seem to work on Windows. I don't have a Linux box set up with Kerberos at the moment, but I was considering trying to get one set up so that I could add support. Seems like a lot of folks that use AD don't actually set up the Kerberos stack on Linux; they just use it for LDAP authn/authz, which doesn't allow for SSO since you never get a Kerberos ticket.